docs/quickstart.md

```diff
@@ -91,6 +91,6 @@ $ draft init
 $ eval $(minikube docker-env)
 ```
 
-The `eval $(minikube docker-env)` command allows Draft to build images directly using Minikube's Docker daemon which lets you skip having to set up a remote/external container registry.
+The `eval $(minikube docker-env)` command allows Draft to build images directly using Minikube's Docker daemon which let's you skip having to set up a remote/external container registry.
```

Comment:
The original spelling was correct. Let's is a contraction of "let us" whereas lets is the verb. Can you change this back? Thanks. :)